http-api-data.cabal

```diff
@@ -52,7 +52,8 @@ library
                    , time-locale-compat    >= 0.1.1.0  && < 0.2
                    , unordered-containers  >= 0.2.6.0  && < 0.3
                    , uri-bytestring        >= 0.1.7    && < 0.4
-                   , uuid-types            >= 1.0.2    && <1.1
+                   , uuid-types            >= 1.0.2    && < 1.1
+                   , cookie                >= 0.4      && < 0.4.4
```

Comment:
`>= 0.4.4 && <0.4`

Reply:
I changed it to ``>= 0.4      && < 0.45`` or it would complain again asking to set ``- cookie-0.4.3``
in the extra-deps.